scripts/apt/install_runtime_dependencies.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+BUILD_DEPENDS=(curl gnupg2 gpg)
+
+apt-get update
+apt-get install -y ${BUILD_DEPENDS[@]} libdlib19.1 ffmpeg exiftool libheif1
```

Comment:
I hesitate about `sqlite` and `reflex` now. They are developing tools, but not dependencies. Someone could change API codes without `sqlite`. Taking an option for scripts makes them complex.

How about mention those tools in doc?

Reply:
Sure, it would be great to mention the changes and recommendations / typical use cases in the doc.
But my main focus here is on the missing SQLite app in the image, while it is installed there in the `master` branch. As I understand, it is mandatory if the user uses SQLite as the Photoview's DBMS. So, after the current implementation is merged "as is now", this case (using SQLite DB for Photoview) won't work or will be significantly limited.
Do I miss something or misunderstand the dependency?

Reply:
`sqlite` is an in-process database, which compiles into the `api` binary. It doesn't require any other libraries to operate the database file.

To check data, it usually installs `sqlite` library to operate the database file outside the `api` binary. That's why I said it's a developing tool, but not a dependency.

You can see the CI passed even without `sqlite3`.

Reply:
Oh, ok. Then, if the Backend doesn't need the SQLite app to be installed for work, sure, we don't need to install it in the final image, as this is a dead app in this case (not used by Photoview, wasting some space, and potential attacking vector)